static/src/javascripts/projects/common/modules/ui/autoupdate.js

```diff
@@ -176,7 +176,8 @@ define([
 
             this.notificationBar = new NotificationBar({attachTo: $('.js-update-notification')[0] });
 
-            $(options.attachTo).addClass('autoupdate--has-animation');
+            var identity = function (val) { return val; };
+            $(options.attachTo.filter(identity)).addClass('autoupdate--has-animation');
```

Comment:
As discussed in person, `options.attachTo` is an array of `node | undefined`. The error occured when you tried calling `addClass` on a bonzo element that wrapped `undefined`. Here we filter out the `undefined` values.

I don't like the idea of `attachTo`—these elements would be much better explicitly passed in as options rather than as a list—but now's not the time for a refactor…